src/utils/bytes.ts

```diff
@@ -2,6 +2,13 @@ import BN from 'bn.js';
 import crypto from 'crypto';
 import { hexToBytes } from 'ethereum-cryptography/utils';
 import { BytesData } from '../models/formatted-types';
+import { isReactNative } from './runtime';
+
+// TextEncoder/TextDecoder (used in this file) needs to shimmed in React Native
+if (isReactNative) {
+  // eslint-disable-next-line global-require
+  require('fast-text-encoding');
```

Comment:
There's no risk of "double polyfilling" because fast-text-encoding checks whether TextEncoder exists before polyfilling it.